<?xml version="1.0"?>
<launch>
    <arg name="videoConfig" />
    <arg name="bagFilepath" />
    <arg name="videoFilepath" />

    <node name="display_cameras" pkg="navigation_test_analysis" type="displayCameras.py" output="screen" />

    <node name="screen_recorder" pkg="navigation_test_analysis" type="screenRecorder.py" output="screen">
        <param name="~bagFilepath" value="$( arg bagFilepath )" />
        <param name="~videoFilepath" value="$( arg videoFilepath )" />
        <rosparam command="load" file="$( arg videoConfig )" />
    </node>
</launch>
